Dockerfile

```diff
@@ -74,6 +73,7 @@ COPY setup.py setup.py
 # Caching models when building the image makes a dockerized server start up faster, but is slow for
 # running tests and things, so we skip it by default.
 ARG CACHE_MODELS=false
+COPY scripts/cache_models.py scripts/cache_models.py
```

Comment:
Why did you move where in the docker build the file is copied? 
Can it not take place on line 65?

Reply:
Oh, did line 65 erase the result of making `EVALB`?  🤦‍♂️   Sorry...

The only issue with doing it this way is that we won't have all of the scripts.  That's probably not an issue...?  I guess this seems fine for now, and if someone wants a particular script in the docker image that's not present, they can add it later.

@murphp15, the previous line copied an entire directory.  Turns out that caused some problems, because of the earlier lines.  Most anticipated uses of the docker container don't actually need all of the scripts in the directory, so to avoid clobbering the stuff built on line 57, we can just copy over a few scripts that we need instead of the whole thing.

Reply:
Actually, I'm now confused about how this is passing CI.  TeamCity runs `./scripts/mypy.sh` and `./scripts/pylint.sh`, but it sure doesn't look like those get copied to the docker image anymore.  What's going on here?  How did those actually run?

Reply:
I found it: TeamCity mounts the VCS root in the Docker image and runs its operations in that directory.  I'd like to change this so it runs its commands from within the Docker image itself, which would require us to have the scripts present in the Docker image.